src/Utils/Result.php

```diff
@@ -33,6 +33,14 @@ public function __construct(User $user, float $distance = null)
         $this->distance = $distance;
     }
 
+    /**
+     * @return int
+     */
+    public function getUserId(): int
```

Comment:
```suggestion
    public function getId(): int
```
Pas besoin de préciser que c'est de l'user, il faut en faire abstraction pour le DTO